components/camera/collectors.go

```diff
@@ -20,6 +21,7 @@ type method int64
 const (
 	nextPointCloud method = iota
 	readImage      method = iota
+	readImages     method = iota
```

Comment:
[Super-Nit] I think we only need the first one to have `method = iota`, I know the second one was here already but I think we should remove it.

Reply:
Good catch.